lib/replicate.js

```diff
@@ -262,17 +262,23 @@ function replicate(repId, src, target, opts, returnValue, result) {
     }
     return src.allDocs({
       keys: ids,
-      include_docs: true
+      include_docs: true,
+      attachments: true
     }).then(function (res) {
       if (state.cancelled) {
         completeReplication();
         throw (new Error('cancelled'));
       }
       res.rows.forEach(function (row) {
+        var hasAttachments = row.doc && row.doc._attachments &&
+          Object.keys(row.doc._attachments).length > 0;
+        var hasAttachmentData = hasAttachments &&
+          !row.doc._attachments[Object.keys(row.doc._attachments)[0]].stub;
+
```

Comment:
This is to support CouchDB <1.6. Support for `attachments=true` in views [was added in 1.6](https://issues.apache.org/jira/browse/COUCHDB-1923).